bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/core/mapping/FieldToContractInputMapping.java

```diff
@@ -59,6 +59,8 @@ public ContractInput toContractInput() {
         final ContractInput contractInput = ProcessFactory.eINSTANCE.createContractInput();
         contractInput.setName(field.getName());
         contractInput.setType(toContractInputType());
+        contractInput.setMultiple(field.isCollection());
```

Comment:
Becarefull field.isCollection() can be null (here we a Boolean not a boolean, you may handle null use case (with a default value)